frontend/viewdir/jobRolePage.njk

```diff
@@ -3,6 +3,30 @@
 {% block content %}
     <div class="container">
         <h3 id="roleName" style="color:#575756">{{role.roleName}}</h3>
-        <h5><span style="color:#41679f">Band Level: </span><span id="bandName" style="color:#575756">{{band.bandName}}</span></h5>
+
+        <div class="row">
+            <h6>
+                <div class="col" id="roleDatePosted">Date Posted: {{specification.datePosted}}</div>
+                <div class="col" id="roleHours">{{specification.hours}}</div>
+                <div class="col" id="roleLocation">{{specification.location}}</div>
+            </h6>
+        </div>
+
+        <div class="divider"></div>
+
+        <h5>
+            <span style="color:#41679f">Band Level: </span><span id="bandName" style="color:#575756">{{band.bandName}}</span></h5>
+
+        <div class="description">
+            <div class="divider"></div>
+            <div class="section">
+                <h5><span style="color:#41679f">Role Description </span></h5>
+                <p id="description">{{specification.roleDesc}}</p>
+                {% if specification.link !== "" %}
```

Comment:
What if `specification.link` is undefined? You might need to amend this slightly to
```suggestion
                {% if specification.link and specification.link !== "" %}
```